### PR TITLE
[bitnami/contour] Adding CRDs for Contour 1.23

### PR DESCRIPTION
**Pull request by langecode**
### Description of the change

Upgrading CRDs to match Contour version 1.23.

Essentially just doing `yq -s '.spec.names.plural'` on CRDs from https://github.com/projectcontour/contour/blob/v1.23.0/examples/contour/01-crds.yaml

### Benefits

Allow for features of 1.23 to be used.

### Possible drawbacks

N/A

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

Signed-off-by: Thor Anker Kvisgård Lange <tal@netic.dk>

### CODE REVIEW

File: bitnami/contour/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/containers/tree/main/bitnami/contour
   - https://projectcontour.io
-version: 10.0.0
+version: 10.0.1
```
Comment (jotamartos) [inline bitnami/contour/Chart.yaml:28]: Please update the chart to 10.1.0. This adds new functionalities to the chart

File: bitnami/contour/resources/contourconfiguration.yaml [hunk 2]
```diff
@@ -1,8 +1,10 @@
+---
```
Comment (jotamartos) [inline bitnami/contour/resources/contourconfiguration.yaml:1]: Please remove this first line in all the files you edited.

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thanks your your contribution. I just added some comments for you to check

**Comment by langecode** [comment]
Bumb. Review comments implemented a while ago.

**Comment by jotamartos** [comment]
Sorry for the delay, I didn't get any notification regarding your update. Changes LGTM but there are some conflicts now. Can you upgrade your base branch and resolve the conflicts? 

Thanks

**Comment by langecode** [comment]
Yup, merged from upstream `main` now.

**Review by jotamartos — APPROVED** [review]

**Pull request merged**